### ISSUE TITLE
bot broken

### ISSUE THREAD

**Issue by BAAAKA** [Jannik66/WALL-E#8]
bot not work on pr1sm, please fix fast

**Comment by BAAAKA** [comment]
@GiyoMoon Fix where

**Comment by GiyoMoon** [comment]
Get 100 Accessibility and SEO rank on https://doremy.ch/ and https://niko.dance/. Then I'll fix it :)

**Comment by BAAAKA** [comment]
Is 100 SEO Score good enough? 
Dont care about Accessibility စ__စ

**Comment by GiyoMoon** [comment]
👩‍🦽

Look at me, how am I supposed to access these sites with my immobility? :( Don't you care about every single user?

**Comment by BAAAKA** [comment]
Fair enough, both https://doremy.ch/ and https://niko.dance/ should be enjoyed by everyone, including you.

**Comment by GiyoMoon** [comment]
Happy to hear! See you at score 100 😎
